src/tests/TokenTests.t.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+import "../TokenFuzzTests.sol";
+import "./SampleToken.sol";
+
+contract TokenTestsTests is TokenTests, TokenFuzzTests 
+ {
+    function setUp() public {
+        _token_ = address(new SampleToken());
+        _tokenName_ = "Sample Token";
+        _contractName_ = "SampleToken";
+        _symbol_ = "TKN";
+
+        TokenLike(_token_).deny(address(this)); // this is not necessary and only done here to check that tests are not expecting the test contract to be auth
+    }
+
+    function test_bulkCheckMintBurn() public {
```

Comment:
Can we also add the fuzzing ones to make sure all is working?

Reply:
Done